applications/nublado/values.yaml

```diff
@@ -725,15 +804,13 @@ cronjob:
     # -- Repository volume definition
     # @default -- See `values.yaml`
     targetVolume:
+      # -- Name of volume to mount (from controller.lab.config.volumes)
+      # @default -- None, must be set for each environment
+      volumeName: null
```

Comment:
Should the artifacts section match the tutorials?

Reply:
In practice yes but not necessarily.  I think.

Reply:
Yes, it should.